packages/augur-ui/src/modules/account/components/notifications.tsx

```diff
@@ -368,7 +374,7 @@ const Notifications = ({ toggle }: NotificationsProps) => {
       toggle={toggle}
       customClass={classNames({
         [Styles.DarkBackgroundMobile]: notificationCount !== 0,
-        [Styles.NoBorderAndBgOnMobile]: notificationCount === 0,
+        [Styles.NoBorderAndBgOnMobile]: notificationCount !== 0,
```

Comment:
if these are both going to get applied at the same time every-time (same condition to apply it) then why don't we just drop one of these classes and combine their definitions?

Reply:
Good one. Done.